packages/firestore/android/src/main/java/io/invertase/firebase/firestore/UniversalFirebaseFirestoreModule.java

```diff
@@ -24,10 +24,12 @@
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.tasks.Tasks;
 import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.LoadBundleTask;
 import io.invertase.firebase.common.UniversalFirebaseModule;
 import io.invertase.firebase.common.UniversalFirebasePreferences;
 import java.util.Map;
 import java.util.Objects;
+import java.nio.charset.StandardCharsets;
```

Comment:
I used to have problems with this one! The minSdkVersion in Android that has these symbols is 19 and luckily that's the firestore package minimum right now, so this is not a breaking change. Whew :sweat_smile: